static/admin/config.yml

```diff
@@ -32,3 +32,20 @@ collections:
               - { name: "subHeadline", widget: "string" }
               - { name: "metaTitle", widget: "string" }
               - { name: "content", widget: "markdown" }
+  - name: "faqEntry"
+    label: "FaqEntry"
+    folder: "src/data/faqEntry"
+    create: true
+    format: "json"
+    fields:
```

Comment:
This declares a collection (=content type) named `faqEntry`, which is a so called [folder collection](https://www.netlifycms.org/docs/collection-types/#folder-collections). Each item is an object with the specified `fields`. One json file will be created in `src/data/faqEntrỳ` per content item.

These options are documented [here](https://www.netlifycms.org/docs/configuration-options/#collections)